pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -64,7 +65,7 @@ let
           ++ lib.optional (cfg.enableFXCastBridge or false) fx_cast_bridge
           ++ extraNativeMessagingHosts
         );
-      libs =   lib.optional stdenv.isLinux udev
+      libs =   lib.optionals stdenv.isLinux [ udev libva ]
             ++ lib.optional ffmpegSupport ffmpeg
```

Comment:
Shouldn't that be the same ffmpeg version we are using in the build?

Reply:
I was already wondering about this, because only the wrapper depends on `ffmpeg`. Do we build Firefox with a bundled version from Mozilla? Or did I miss something?

Apart from that `ffmpeg_4` would be nice to have in general :) E.g. Arch also builds with 4.x

Reply:
I asked in #build:mozilla.org. Apparently they don't require an ffmpeg to build (which is true, based on our derivation). My guess is the thing in tree is a wrapper, but I didn't look; I'm afraid of falling down a rabbit hole.